service/src/test/java/com/epam/esm/service/impl/CertificateServiceImplTest.java

```diff
@@ -0,0 +1,121 @@
+package com.epam.esm.service.impl;
+
+import com.epam.esm.dto.certificateDto.ReadCertificate;
+import com.epam.esm.entity.Certificate;
+import com.epam.esm.mapper.impl.certificateMapper.TransitionCertificateFromCertificateDto;
+import com.epam.esm.mapper.impl.certificateMapper.TransitionCertificateFromCreateCertificate;
+import com.epam.esm.mapper.impl.certificateMapper.TransitionReadCertificateFromCertificate;
+import com.epam.esm.repository.CertificateRepository;
+import com.epam.esm.repository.TagRepository;
+import com.epam.esm.util.messange.LanguageMassage;
+import com.epam.esm.util.validator.impl.CertificateValidator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
avoid imports with *

Reply:
Fixed.
The dependency was added automatically when the test was created.